Client/mods/deathmatch/logic/luadefs/CLuaObjectDefs.cpp

```diff
@@ -436,9 +436,10 @@ int CLuaObjectDefs::SetObjectScale(lua_State* luaVM)
         argStream.ReadNumber(vecScale.fY, vecScale.fX);
         argStream.ReadNumber(vecScale.fZ, vecScale.fX);
     }
+    argStream.ReadBool(forceUpdate, false);
```

Comment:
I think there should be a way to get variable `eventName` from current LuaVM. This will remove additional argument from API

Reply:
You can, its a global.
```cpp
lua_getglobal(luaVM, "eventName");
std::string_view eventName = lua_tostring(luaVM, -1);
// Not sure if it needs to be popped: lua_pop(luaVM);
```